src/components/legend.ts

```diff
@@ -11,6 +11,10 @@ export module Component {
      * The css class applied to each legend entry
      */
     public static LEGEND_ENTRY_CLASS = "legend-entry";
+    /**
+     * The css class applied to each legend symbol
+     */
+    public static LEGEND_SYMBOL_CLASS = "legend-symbol";
```

Comment:
We should note that people should update their CSS selectors for this from `<circle>` to `.legend-symbol`.